.github/workflows/e2e.yaml

```diff
@@ -17,6 +17,12 @@ jobs:
         # Latest patch version can be found in https://github.com/kubernetes/website/blob/main/content/en/releases/patch-releases.md
         # Some versions might not be available yet in https://storage.googleapis.com/kubernetes-release/release/v1.X.Y/bin/linux/amd64/kubelet
         k8sVersion: [ "v1.22.3", "v1.21.4", "v1.20.10", "v1.19.14", "v1.18.20", "v1.17.17", "v1.16.15" ]
+        cri: [ docker ]
+        include:
+          - k8sVersion: v1.22.3
+            cri: containerd
+          - k8sVersion: v1.21.4
+            cri: containerd
```

Comment:
There is no real reason to limit containerd to these versions. I just felt doubling the amount of jobs was a bit too much.

Reply:
I agree